src/text-editor.coffee

```diff
@@ -114,6 +116,12 @@ class TextEditor extends Model
       @emit 'scroll-left-changed', scrollLeft
       @emitter.emit 'did-change-scroll-left', scrollLeft
 
+    @gutterContainer = new GutterContainer this
+    @gutterContainer.addGutter
+      name: LineNumberGutterName
```

Comment:
I think that `name` should be `gutterName` because otherwise it's unclear whether we're naming the decoration itself or the gutter it belongs to.